apache-Dockerfile-block-1

```diff
@@ -5,7 +5,7 @@ RUN rm -rf /var/www/html && mkdir -p /var/lock/apache2 /var/run/apache2 /var/log
 # Apache + PHP requires preforking Apache for best results
 RUN a2dismod mpm_event && a2enmod mpm_prefork
 
-RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist
+RUN mv /etc/apache2/apache2.conf /etc/apache2/apache2.conf.dist && rm /etc/apache2/{conf,sites}-enabled/*
```

Comment:
Right, but it'll be pretty frustrating if at some point we need to compare to what's enabled by default and the symlinks are just completely gone, won't it?  That's the rationale for keeping the original `apache2.conf` around, and because it makes it clear we've made a change from the original default.